README.md

```diff
@@ -55,7 +55,10 @@ More resources if you get stuck:
 
 ### How to Build
 
-Rhino builds with `Gradle`. Here are some useful tasks:
+Rhino builds with `Gradle`. You don't need to install `Gradle` locally to build `Rhino` thanks to [Gradle wrapper](http://gradle.org/docs/current/userguide/gradle_wrapper.html)
+Just clone project from `GitHub` and use provided `gradlew.sh` or `gradlew.bat` for `Linux` / `Mac OS` or `Windows` accordingly.
```

Comment:
Minor correction: even though file for *nix is a bash script, it's called just `gradlew`, not `gradlew.sh`.